lib/services/runtime/handlers/capture/capture.alexa.ts

```diff
@@ -0,0 +1,81 @@
+import { AlexaNode } from '@voiceflow/alexa-types';
+import { VoiceflowConstants } from '@voiceflow/voiceflow-types';
+import { Intent } from 'ask-sdk-model';
+import _ from 'lodash';
+import wordsToNumbers from 'words-to-numbers';
+
+import { HandlerFactory } from '@/runtime';
+import { Turn } from '@/runtime/lib/constants/flags.alexa';
+
+import { RequestType } from '../../types.alexa';
+import { addRepromptIfExists, mapSlots } from '../../utils.alexa';
+import CommandHandler from '../command/command.alexa';
+import RepeatHandler from '../repeat/repeat.alexa';
+// import { TurnElicitSlot } from './responseBuilders';
+// import { createElicitSlot } from './utils/directives';
+
+const getSlotValue = (intent: Intent) => {
+  const intentSlots = intent.slots || {};
+  const value = Object.keys(intentSlots).length === 1 && Object.values(intentSlots)[0]?.value;
+  if (!value) return null;
+
+  const num = wordsToNumbers(value);
+  if (typeof num !== 'number' || Number.isNaN(num)) {
+    return value;
+  }
+  return num;
+};
+
+const utilsObj = {
+  mapSlots,
+  getSlotValue,
+  addRepromptIfExists,
+  commandHandler: CommandHandler(),
+  repeatHandler: RepeatHandler(),
+};
+
+export const CaptureAlexaHandler: HandlerFactory<AlexaNode.Capture.Node, typeof utilsObj> = (utils) => ({
```

Comment:
I think we can just use `import { VoiceNode } from '@voiceflow/voice-types'; `
```suggestion
export const CaptureAlexaHandler: HandlerFactory<VoiceNode.Capture.Node, typeof utilsObj> = (utils) => ({
```
Given the same data format, it will allow us to modify the runtime in the future.